atd-vzd/hasura/hasura_metadata_2023_04_11_14_03_58_508.json

```diff
@@ -0,0 +1,5527 @@
+{
+  "version": 2,
```

Comment:
I noticed the existing metadata is version 3. Should export metadata from Hasura v1 which is in production or v2 which is the local stack? Happy to take this back to version 3 if that is what we've been doing.

Reply:
this was my doing from exporting metadata from my local stack a few weeks ago, i didnt realize there was a diff in versions btwn local and prod. imo it makes sense to export the prod version though, so my bad!

Reply:
all good! just gathering opinions because I can see it both ways. 🙏